imap_processing/cdf/utils.py

```diff
@@ -63,6 +69,8 @@ def load_cdf(file_path: Path, **kwargs: dict) -> xr.Dataset:
     dataset : xr.Dataset
         The ``xarray`` dataset for the CDF file
     """
+    if "to_datetime" not in kwargs:
+        kwargs["to_datetime"] = True
```

Comment:
Can we add a comment in the function doc string that we are setting this by default? Otherwise, it looks great to me

Reply:
Nice improvement to this function!